tests/kernel/run_ltp.pm

```diff
@@ -266,6 +266,7 @@ sub run {
         );
 
         script_run('ps axf');
+        script_run('netstat -ap');
```

Comment:
Apparently this is deprecated https://lwn.net/Articles/710533/#Comments

On my system `ss -ap` executes a lot faster although the output may be different.